feugy/tabulous
# feat(chess): 🎅 introducing chess! 🎅

### :book: What's in there?

Ho ho ho!

This is Santa's special delivery: a rule-less Chess game with hand-made 3D meshes.
Merry Christmas and happy new year!

- fix(web): clicking on mesh detail overlay also clicks in the engine
- feat(chess): introduces chess game!
- feat(web): semi-transparent indicators, made opaque when hovering their mesh
- chore(web): supports STL mesh files
- chore(web): includes babylon inspector's peer dependencies

### :test_tube: How to test?

Enjoy the new game!

For testing mesh detail overlay:
1. create a Klondike game
2. double-click on any card
3. click on on any other card to close the mesh details overlay
   > This should not flip the "clicked" card, only close the overlay

Here's an example of semi-transparent indicators:

[transparent-indicators.webm](https://user-images.githubusercontent.com/186268/209585820-530e1ebd-33af-41e0-8547-f27d440a757e.webm)
